Add 4 new authors to authors.yml

havi opened the pull request:
TODO: 
- [ ] Add John's avatar (done but pixilated)
- [x] Add Alex's avatar
- [x] Add Olabayo avatar
- [x] Add Lukonde avatar

Current images lifted from Twitter and LinkedIn, need to confirm these are correct:

havi committed:
Add 3 new authors to authors.yml

mraible commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+alex-doukas:
+  full_name: "Alex Doukas"
+  display_name: "Alex Doukas"
+  avatar:  
```
Missing avatar.

havi replied:
yup. that's why i saved as draft. waiting for avatar.

mraible commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
   bio: I ❤️ all things tech, open source and community. Developer 🥑 for @GraalVM at @OracleLabs, helping @jhipster, ex @googledevgroups
+  
+alex-doukas:
+  full_name: "Alex Doukas"
```
Quotes aren't needed around these values. If you look at the others, they work just fine without quotes.

havi replied:
```suggestion
  full_name: Alex Doukas
```

havi commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+  bio: I'm Alex, a front end developer. My main area of expertise is web development and everything that comes along with it. I also have extensive knowledge of topics such as software testing, UX design, big data, social media marketing, and SEO techniques.
+  
+john-gramila:
+  full_name: "John Gramila"
```
```suggestion
  full_name: John Gramila
```

havi commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+  
+john-gramila:
+  full_name: "John Gramila"
+  display_name: "John Gramila"
```
```suggestion
  display_name: John Gramila
```

havi commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+  bio: John is a developer and writer working in Chicago. Failure and success both create growth, and growth never stops.
+
+lukonde-mwila:
+  full_name: "Lukonde Mwila"
```
```suggestion
  full_name: Lukonde Mwila
```

havi commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+
+lukonde-mwila:
+  full_name: "Lukonde Mwila"
+  display_name: "Lukonde Mwila"
```
```suggestion
  display_name: Lukonde Mwila
```

havi commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,33 @@ alina-yurenko:
+  twitter: https://twitter.com/LuKE9ine
+  bio: Lukonde is a Principal Technical Evangelist at SUSE and is an AWS Container Hero. He specializes in cloud and DevOps engineering and cloud-native technologies. He is passionate about sharing knowledge through various mediums and engaging with the developer community at large.
+
+
```
```suggestion
olabayo-balogun:
  full_name: Olabayo Balogun
  display_name: Olabayo Balogun
  avatar: avatar-OBalogun.jpg
  linkedin: https://www.linkedin.com/in/olabayobalogun/
  bio: "A .Net, Blazor and Angular software engineer who is committed to building technological infrastructures capable of solving the challenges of netizens now and in the future. "
  

```
one more bio

bdemers committed:
Add avatar for Lukonde Mwila

Copied from twitter, the original source file needed we had needed to be cropped, but the version of this file used on Lukonde's twitter profile was already cropped square

bdemers committed:
Apply suggestions from code review

Co-authored-by: havi hoffman <havi.hoffman@gmail.com>

bdemers committed:
Apply suggestions from code review

Co-authored-by: havi hoffman <havi.hoffman@gmail.com>

bdemers committed:
Added avatar for JGramila

bdemers committed:
Added avatar for OBalogun

bdemers committed:
Added avatar for ADoukas

havi committed:
Merge branch 'main' into new-draftdev-authors

bdemers committed:
adjust size of ADoukas avatar

bdemers commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,40 @@ alina-yurenko:
+  
+  
+  
+  
```
```suggestion
  bio: "A .Net, Blazor and Angular software engineer who is committed to building technological infrastructures capable of solving the challenges of netizens now and in the future. "

```

bdemers committed:
Update _source/_data/authors.yml

bdemers commented on _source/_data/authors.yml:
```diff
@@ -802,3 +802,34 @@ alina-yurenko:
+alex-doukas:
+  full_name: Alex Doukas
+  display_name: "Alex Doukas"
+  avatar:  
```
```suggestion
  avatar:  avatar-ADoukas.jpg
```

bdemers committed:
Apply suggestions from code review

bdemers committed:
Updating ADoukas avatar with generic initials

havi committed:
Merge branch 'main' into new-draftdev-authors

bdemers committed:
Merge branch 'main' into new-draftdev-authors
